docs/source/application_development/testnet.rst

```diff
@@ -0,0 +1,64 @@
+=============
+Lynx Testnet
+=============
+
+NuCypher provides a public Long-Term Support testnet running on the ethereum Goerli testnet as a stable playground
+for application development and network users (e.g., Alices wishing to create sharing policies, grant & Retrieve, etc.).
+
+.. note::
+
+    Lynx testnet tokens are not needed to use nucypher as Alie, Bob, or Enrico.
+    If you'd like to volunteer to run an Lynx testnet node, reach out to the
+    team on our `Discord server <https://discord.gg/7rmXa3S>`_.
+
+
+.. important::
+
+    Goerli ETH is required to use the Lynx testnet.
+
+
+Alice and Bob work support the lynx testnet using the python API:
```

Comment:
not sure what is meant here.

Reply:
Superfluous "work" - Thanks for noticing.